src/Symfony/Component/Mailer/Bridge/Amazon/composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": ">=7.2.5",
         "symfony/deprecation-contracts": "^2.1",
-        "symfony/mailer": "^5.2"
+        "symfony/mailer": "^4.4|^5.0"
```

Comment:
changing the constraint to `^4.4.12|^5.1.4` should help as it only allows to install releases of the Mailer component that contain #37888

Reply:
Done.